app/views/users/_profile_cumulativestats.html.haml

```diff
@@ -0,0 +1,46 @@
+%h3
+  = t("users.contribution_statistics")
+%br/
+%h5
+  = t("users.user_impact")
+  #{@user.username}
+.profile-stats
+  .stat-display
+    .stat-display__stat
+      .stat-display__value= presenter.courses.count
+      %small= t("#{presenter.course_string_prefix}.courses_taught")
+    .stat-display__stat.tooltip-trigger
+      .stat-display__value= presenter.user_count
+      %small= t("#{presenter.course_string_prefix}.students")
+      .tooltip.dark
+        %h4
+          = number_to_human presenter.trained_percent
+          \%
+        %p= t("users.up_to_date_with_training")
+%br/
+%br/
```

Comment:
better to use css within the users module than handle spacing with empty elements and breaks.